### PR TITLE
Feat/nomenclature resources

### PR DESCRIPTION
**Pull request by folland87**

### CODE REVIEW

File: src/api/app.js [hunk 1]
```diff
@@ -64,7 +67,11 @@ app.use(authenticate);
 app.use(assetsRoutes);
 app.use(categoriesRoutes);
 app.use(documentsRoutes);
+app.use(documentTypesRoutes);
+app.use(emailTypesRoutes);
+app.use(documentsRoutes);
```
Comment (annelhote) [inline src/api/app.js:72]: it seems that `app.use(documentsRoutes);` is duplicated (cf. l. 69).

File: src/api/categories/__tests__/categories.test.js [hunk 2]
```diff
@@ -0,0 +1,240 @@
+import { categories as resource } from '../../resources';
+
+let authorization;
+let id;
+let textId;
+
+const payload = {
+  usualNameFr: 'Catégori',
```
Comment (annelhote) [inline src/api/categories/__tests__/categories.test.js:8]: I would enjoy to have `Catégorie` instead of `Catégori`.

**Review by annelhote — CHANGES_REQUESTED** [review]

File: src/api/categories/__tests__/categories.test.js (hunk 2 above)
Comment (folland87) [reply src/api/categories/__tests__/categories.test.js:8]: Haha it's a mistake! so it's updated with the patch route :D

Comment (annelhote) [reply src/api/categories/__tests__/categories.test.js]: Oh I see. Too smart for me :smile:

**Review by annelhote — APPROVED** [review]

**Pull request merged**